src/commonMain/kotlin/spaceEngineers/controller/SpaceEngineers.kt

```diff
@@ -0,0 +1,33 @@
+package spaceEngineers.controller
+
+import spaceEngineers.model.SeObservation
+import spaceEngineers.model.ToolbarLocation
+import spaceEngineers.model.Vec3
+
+interface SpaceEngineers {
+    val session: Session
+    val character: Character
+    val items: Items
+    val observer: Observer
+}
+
+interface Session : WorldController
+
+interface Character {
+    fun moveAndRotate(movement: Vec3 = Vec3.ZERO, rotation3: Vec3 = Vec3.ZERO, roll: Float = 0f): SeObservation
+}
+
+interface Observer {
+    fun observe(): SeObservation
+    fun observeBlocks(): SeObservation
+    fun observeNewBlocks(): SeObservation
+    fun observeEntities(): SeObservation
+}
```

Comment:
What's the idea of separating Observer from Character?

Reply:
Main reason was to copy architecture of server, where observer and charactercontroller are 2 separate classes, but I also like that methods related to each other are grouped instead of having loong list of "all" methods.

Similarly I was thinking, that maybe Items should be part of Character, but then I decided to keep those 2 separated.